styles/Vocab/Plone/accept.txt

```diff
@@ -1,10 +1,22 @@
+async
+api 
+backend
+css
+http 
+https 
+JavaScript
+LTS 
+npm
+nvm 
+NodeJS
+Pastanaga
 `plone.restapi`
 `plone.volto`
-npm
 Plone
 Razzle
 RichText
+repo 
```

Comment:
Remove. "repo" is fine in conversation, but in writing we want to spell out the word as "repository".

```suggestion
```